ome_zarr/reader.py

```diff
@@ -315,6 +328,30 @@ def array(self, resolution: str, version: str) -> da.core.Array:
         # data.shape is (t, c, z, y, x) by convention
         return self.zarr.load(resolution)
 
+    def validate(self, warnings: bool = False) -> None:
+        multiscales = self.lookup("multiscales", [])
+        version = multiscales[0].get("version", "0.1")
+        print("Validating Multiscales spec at", self.zarr)
+        print("Using Multiscales schema version", version)
+        image_schema = get_schema(version)
+
+        # Always do a validation with the MUST rules
+        # Will throw ValidationException if it fails
+        json_data = self.zarr.root_attrs
+        validate(instance=json_data, schema=image_schema)
```

Comment:
Minor comment: this is confusing to read as this is called within a `validate` method. Potentially `jsonschema.validate` would clear any ambiguity on the scope.

Reply:
Fixed in b388a3a